src/components/Formatters/useCurrencyFormatter.tsx

```diff
@@ -0,0 +1,18 @@
+export enum CurrencypPosition {
+  BACK = 'de-DE',
+  FRONT = 'ja-JP',
+}
+
+export const useCurrencyFormatter = (
+  options?: Intl.NumberFormatOptions & { currencyFormat?: CurrencypPosition }
+) => {
+  const formatter = new Intl.NumberFormat(
+    options?.currencyFormat ?? CurrencypPosition.FRONT,
```

Comment:
This argument is the locale to format in. We are leaving it `undefined` to use the current locale. In the future we might pull this from context, which is why this logic is in a hook.
```suggestion
    undefined,
```